faunadb-java/src/main/java/com/faunadb/client/query/Language.java

```diff
@@ -2201,6 +2201,19 @@ public static Expr HasIdentity() {
     return Fn.apply("has_identity", Null());
   }
 
+  /**
+   * Returns current token used for the request.
+   *
+   * @return a new {@link Expr} instance
+   * @see <a href="https://app.fauna.com/documentation/reference/queryapi#authentication">FaunaDB Authentication Functions</a>
```

Comment:
maybe we should fix this url to point to the function doc `https://docs.fauna.com/fauna/current/api/fql/functions/<functionname>` or to https://docs.fauna.com/fauna/current/api/fql/cheat_sheet

@nathaniel-may @faunaee thoughts?

Reply:
That definitely looks like a copy+paste of the (now very) old documentation links.

We should point to the location where the function reference page will exist. In this case, that would be https://docs.fauna.com/fauna/current/api/fql/functions/currenttoken

We should only point to the cheat sheet page when we don't have a good idea where to send users.